File: src/main/java/com/itsci/mju/maebanjumpen/partyrole/service/AdminService.java
package com.itsci.mju.maebanjumpen.partyrole.service;

import com.itsci.mju.maebanjumpen.partyrole.dto.AdminDTO;

import java.util.List;

public interface AdminService {
    List<AdminDTO> getAllAdmins();
    AdminDTO getAdminById(int id);

    AdminDTO saveAdmin(AdminDTO adminDto);

    void deleteAdmin(int id);

    AdminDTO updateAdmin(int id, AdminDTO adminDto);
}

File: src/main/kotlin/com/itsci/mju/maebanjumpen/partyrole/service/AdminService.kt
package com.itsci.mju.maebanjumpen.partyrole.service

import com.itsci.mju.maebanjumpen.partyrole.dto.AdminDTO

interface AdminService {
    fun listAllAdmins(): List<AdminDTO>
    fun getAdminById(id: Int): AdminDTO
    fun createAdmin(adminDto: AdminDTO): AdminDTO
    fun deleteAdmin(id: Int)
    fun updateAdmin(id: Int, adminDto: AdminDTO): AdminDTO
}

